build.sbt

```diff
@@ -29,13 +29,16 @@ libraryDependencies ++= {
     "com.typesafe.akka" %% "akka-stream" % Akka,
     "com.typesafe.akka" %% "akka-http" % AkkaHttp,
     "de.heikoseeberger" %% "akka-http-json4s" % AkkaHttpJson4s,
+    "de.heikoseeberger" %% "akka-http-circe" % AkkaHttpJson4s,
```

Comment:
...why this dependency? I'd rather use only 1 json library at the time and we are already using json4s ;)

Reply:
Don't remember exactly; I updated a few Akka dependency versions and `akka-http-circe` somehow became necessary, or so I thought.  Irrelevant now, in any case.